include/velocypack/StringRef.h

```diff
@@ -54,11 +54,7 @@ class StringRef {
   constexpr StringRef(char const* data, std::size_t length) noexcept : _data(data), _length(length) {}
   
   /// @brief create a StringRef from a null-terminated C string
-#if __cplusplus >= 201703
   constexpr explicit StringRef(char const* data) noexcept : StringRef(data, std::char_traits<char>::length(data)) {}
-#else
-  explicit StringRef(char const* data) noexcept : StringRef(data, std::strlen(data)) {}
-#endif
    
```

Comment:
Shouldn't we have also ctor from string_view ? (maybe even implicit as it doesn't not imply data copying)

Also it could be useful to have implicit conversion to string_view operator.
like this:
`  constexpr operator std::string_view() const noexcept {
    return std::string_view(_data, _length);
  }`

This will allow us transparently use StringRef with functions expecting string_view (ideally at some point we could even drop StringRef and use only std stuff)

Reply:
Ok, done.